twigs/fingerprint.py

```diff
@@ -16,7 +16,8 @@
 from . import ssl_audit
 from .dast_plugins import zap as zap_dast
 
-NMAP = "/usr/bin/nmap"
+import subprocess
+NMAP = subprocess.check_output(["which", "nmap"]).strip().decode()
```

Comment:
Consider using shutil.which instead of running subprocess. Also make this check local to where NMAP is actually being used. That way there is no need for a global NMAP. If shutil.which results in a null or error then bail out with an error.